.eslintrc

```diff
@@ -0,0 +1,38 @@
+{
+    "rules": {
+        "indent": [
+            2,
+            2
+        ],
+        "quotes": [
+            2,
+            "single"
```

Comment:
Hilarious that the file that says we use single quotes is in a file that's all double.